Places/Models/Route.h

```diff
@@ -0,0 +1,13 @@
+//
+//  Route.h
+//  Places
+//
+//  Created by Iris Fu on 8/1/22.
+//
+
+#ifndef Route_h
+#define Route_h
+
+typedef NSMutableArray<Place *> Route;
```

Comment:
This is sufficient for a quick test, but maybe it would be worth it defining `Route` as an actual type with origin, destination and waypoints? Up to you...